x-pack/plugins/security_solution/public/common/components/callouts/callout_description.tsx

```diff
@@ -1,26 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import React, { FC } from 'react';
-import { EuiDescriptionList } from '@elastic/eui';
-import { CallOutMessage } from './callout_types';
-
-export interface CallOutDescriptionProps {
-  messages: CallOutMessage | CallOutMessage[];
-}
-
-export const CallOutDescription: FC<CallOutDescriptionProps> = ({ messages }) => {
-  if (!Array.isArray(messages)) {
-    return messages.description;
```

Comment:
NOTE: Removed this file as nothing was sending down messages as an array and instead it was always just hitting the `return messages.description;` here.